src/renderers/dom/client/wrappers/ReactDOMSelect.js

```diff
@@ -58,6 +58,28 @@ function warnIfValueIsNull(props) {
   }
 }
 
+function warnIfDuplicateValues(inst) {
+  const options = inst._currentElement.props.children;
```

Comment:
Also, `children` can be `null` or an element (not wrapped in an array). I'm not sure whether it is supported or not yet, but IIRC the idea is also to support anything that is iterable as children, so we can't assume this to be an array in that case?